sql/item_mods.sql

```diff
@@ -26336,6 +26336,7 @@ INSERT INTO `item_mods` VALUES (21507,8,15);      -- jolt_counter_+1 STR+15
 INSERT INTO `item_mods` VALUES (21507,10,15);     -- VIT+15
 INSERT INTO `item_mods` VALUES (21507,25,30);     -- Accuracy+30
 INSERT INTO `item_mods` VALUES (21507,291,18);    -- Counter+18
+INSERT INTO `item_mods` VALUES (21509,811,25);    -- Final Paradise WS
```

Comment:
Don't think this is right, modifier 811 doesn't exist?

Think you want modID 355 (ADDS_WEAPONSKILL) with a value of the weaponskill it adds (in this case, 228)

Reply:
Oh, is that how this works? I was honestly confused on that.
I will fix that, once I have the WS number figured out xD

Reply:
All fixed now